pkg/poddisruptionbudget/list.go

```diff
@@ -0,0 +1,107 @@
+package poddisruptionbudget
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/golang/glog"
+	"github.com/openshift-kni/eco-goinfra/pkg/clients"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// List returns podDisruptionBudget inventory in the given namespace.
+func List(apiClient *clients.Settings, nsname string, options ...metav1.ListOptions) ([]*Builder, error) {
+	if apiClient == nil {
+		glog.V(100).Infof("The apiClient is empty")
+
+		return nil, fmt.Errorf("podDisruptionBudget 'apiClient' cannot be empty")
+	}
+
+	if nsname == "" {
+		glog.V(100).Infof("podDisruptionBudget 'nsname' parameter can not be empty")
+
+		return nil, fmt.Errorf("failed to list podDisruptionBudgets, 'nsname' parameter is empty")
+	}
+
+	logMessage := fmt.Sprintf("Listing podDisruptionBudget in the nsname %s", nsname)
+	passedOptions := metav1.ListOptions{}
+
+	if len(options) >= 1 {
+		passedOptions = options[0]
+		logMessage += fmt.Sprintf(" with the options %v", passedOptions)
+	}
```

Comment:
if only 1st element is used then better to have explicit check (like in other packages)
```
    if len(options) > 1 {
        glog.V(100).Infof("'options' parameter must be empty or single-valued")

        return nil, fmt.Errorf("error: more than one ListOptions was passed")
    }

    if len(options) == 1 {
        passedOptions = options[0]
        logMessage += fmt.Sprintf(" with the options %v", passedOptions)
    }
```